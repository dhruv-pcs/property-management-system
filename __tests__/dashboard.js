import React from 'react'
import { act, render } from '@testing-library/react'
import '@testing-library/jest-dom'
import Dashboard from 'src/pages/index'

describe('Dashboard Component', () => {
  it('should render dashboard component', async () => {
    const { container } = act(() => render(<Dashboard />))
    expect(container).toMatchSnapshot()
  })
})
