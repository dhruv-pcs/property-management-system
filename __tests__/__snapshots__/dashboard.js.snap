// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component should render dashboard component 1`] = `undefined`;
